src/test_edgepi/unit_tests/test_calibration/test_access_eeprom.py

```diff
@@ -21,6 +24,13 @@ def fixture_test_dac(mocker):
     mocker.patch("edgepi.peripherals.i2c.I2C")
     yield EdgePiEEPROM()
 
+# Max data size of osensa space
+OSENSA_DATA_SIZE = EdgePiMemoryInfo.USER_SPACE_START_BYTE.value-3
```

Comment:
what is 3 :)

Reply:
Osensa reserved space also uses the first two bytes to represents the used data. `OSENSA_DATA_SIZE` is a dummy data size to test memory bound edge case. 
size of `EdgePiMemoryInfo.USER_SPACE_START_BYTE.value-3` should fit in Osensa reserved space with a room for 1 more byte. Adding 1 more byte makes the memory space full. Adding 2 more bytes raises the MemoryOutOfBound Error